### PR TITLE
feat: extract from bip21 follow up

### LINKED ISSUE #302: Allow BIP21 strings
Web app should allow and handle (maybe just allow as valid input, don't visually truncate anything) BIP21 `bitcoin:` strings.

- [ ] Allow regular `bitcoin:` BIP21 as valid address input for reverse swaps, `bitcoin:bc1address`
- [ ] Allow `bitcoin:` as valid invoice input for normal swaps if it is a https://bitcoinqr.dev/ with a valid ln invoice
- [ ] `liquidnetwork:` for liquid addresses
- [ ] `lightning:` for lightning invoices

### PR DESCRIPTION
**Pull request by dni**
closes #302
closes #304 

- added `liquidnetwork:`

### CODE REVIEW

**Review by kilrau — CHANGES_REQUESTED** [review]
see comment above

**Review by kilrau — APPROVED** [review]
Yup, now behaves as expected. LGTM

File: src/utils/invoice.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ type LnurlCallbackResponse = {
 };
 
 export const invoicePrefix = "lightning:";
+export const bitcoinPrefix = "bitcoin:";
+export const liquidPrefix = "liquidnetwork:";
```
Comment (michael1011) [inline src/utils/invoice.ts:22]: `liquidtestnet` for testnet

**Review by michael1011 — CHANGES_REQUESTED** [review]

**Review by michael1011 — CHANGES_REQUESTED** [review]
image

When entering bip21 first, the button shows an error. But the address is valid. Error goes away as soon as the amounts are changed

**Review by michael1011 — CHANGES_REQUESTED** [review]
image

Prefixes of invoices are not removed before sending them to the backend

**Review by michael1011 — APPROVED** [review]

**Pull request merged**